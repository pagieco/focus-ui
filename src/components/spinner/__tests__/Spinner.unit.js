import { mount } from '@vue/test-utils';
import { Spinner } from '../index';

describe('Spinner', () => {
  it('is a valid vue component', () => {
    expect(Spinner).toBeAComponent();
  });

  it('matches the snapshot', () => {
    expect(mount(Spinner).element).toMatchSnapshot();
  });
});
