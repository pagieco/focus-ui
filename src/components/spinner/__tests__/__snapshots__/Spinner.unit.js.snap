// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner matches the snapshot 1`] = `
<div
  class="spinner"
  role="status"
>
  <svg
    height="20"
    stroke-width="16.00"
    viewBox="-3.00 -3.00 106.00 106.00"
    width="20"
  >
    <path
      class="spinner__track"
      d="M 50,50 m 0,-45 a 45,45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
    />
     
    <path
      class="spinner__head"
      d="M 50,50 m 0, -45 a 45, 45 0 1 1 0,90 a 45,45 0 1 1 0,-90"
      pathLength="280"
      stroke-dasharray="280 280"
      stroke-dashoffset="210"
    />
  </svg>
</div>
`;
